### ISSUE TITLE
Could not load file or assembly 'FSharp.Core, Version=4.3.1.0

### ISSUE THREAD

**Issue by PvlD** [fsprojects/FSharp.Data#1232]
VS2017  , the same issue on  Windows 10 , Windows Server 2016 ,  Windows Server 2019,

![image](https://user-images.githubusercontent.com/10778065/49021777-65fcfb00-f161-11e8-8dfb-32231e8af635.png)

($exception).FusionLog:

=== Pre-bind state information ===
LOG: DisplayName = FSharp.Core, Version=4.3.1.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a
 (Fully-specified)
LOG: Appbase = file:///C:/Program Files (x86)/Internet Explorer/
LOG: Initial PrivatePath = NULL
Calling assembly : FSharp.Data, Version=3.0.0.0, Culture=neutral, PublicKeyToken=null.
===
LOG: This bind starts in LoadFrom load context.
WRN: Native image will not be probed in LoadFrom context. Native image will only be probed in default load context, like with Assembly.Load().
LOG: No application configuration file found.
LOG: Using host configuration file: 
LOG: Using machine configuration file from C:\Windows\Microsoft.NET\Framework\v4.0.30319\config\machine.config.
LOG: Post-policy reference: FSharp.Core, Version=4.3.1.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a
LOG: The same bind was seen before, and was failed with hr = 0x80070002.

![image](https://user-images.githubusercontent.com/10778065/49022159-37335480-f162-11e8-8fd0-1195dfad2ad3.png)

**Comment by kmiller77** [comment]
I'm also having this issue. Any fix?

**Comment by PvlD** [comment]
I don't know.  Project was  discontinued.
It was used in IE extension , officially  they not support managed code .

**Comment by fischgeek** [comment]
I'm seeing this issue when I try to `open FSharp.Data.HttpRequestHeaders` and reference it in code. My entire project breaks until I remove references to that specific library. Any thoughts on this?

**Comment by PvlD** [comment]
Sorry,
can't help you.
The issue was not resolved.

**Comment by smoothdeveloper** [comment]
@fischgeek this is probably a binding redirects issue, you may have several F# libraries / projects depending on different versions of FSharp.Core.

https://docs.microsoft.com/en-us/dotnet/framework/configure-apps/redirect-assembly-versions

The general fix is to adjust app.config with a binding redirect declaration unifying the version of FSharp.Core (or any library that clashes) to the one that your application ships.

**Comment by fischgeek** [comment]
@smoothdeveloper - Thanks for the reply. I still think FSharp.Data is using an old version of .Core. I spent a lot of time trying to isolate the issue. 

So, I unloaded all attached F# projects and built the main project (web project). I referenced only 1 F# library project (just a few helper methods). One helper method is using `FSharp.Data.HttpRequestHeaders`. 

With the library as-is running on the latest .Core version, the build succeeds but the site throws the code dump screen. 

When I comment out the 1 line that references `FSharp.Data.HttpRequestHeaders` the site works. 

Interesting, right?
